.rubocop.yml

```diff
@@ -1,20 +1,44 @@
 Rails/CreateTableWithTimestamps:
   Enabled: false
 Rails/HasManyOrHasOneDependent:
-  Enabled: false
+  Enabled: true
+# if a codebase is on an older version on rails override this in the local rubocop.yml
 Rails/HttpPositionalArguments:
-  Enabled: false
+  Enabled: true
```

Comment:
How do we handle older code bases, is overriding locally a reasonable compromise?

Reply:
it seems we can either override or set a target rails version in this yml file https://www.rubydoc.info/gems/rubocop/0.49.1/RuboCop/Cop/Rails/HttpPositionalArguments

Reply:
Can you post the snippet that lets you specify a rails version, I see a ruby version.  The link you sent says to disable to cop if you are on Rails < 5 but not how to flip it if you are greater than 5

Reply:
You may have to try this a little bit, but I think you can do
```ruby
AllCops:
  TargetRailsVersion: 5.0
```
for all cops, and
```ruby
<CopName>:
  TargetRailsVersion: 5.0
```
for a specific cop

Reply:
thanks I will try it out!

Reply:
It looks like Rails/HttpPositionalArguments doesn't respond to `TargetRailsVersion` but you can still set AllCops to it.  Older apps would still need to override this value to set it to `4.0` otherwise the checks with still run.  I think I still recommend us setting this here to true and apps on older rails versions overriding it or setting TargetRailsVersion in their local rubocop.yml file.

Reply:
that seems like a reasonable thing to me